### ISSUE TITLE
docker-compose up from Docker Desktop for Windows giving error for `web` container

### ISSUE THREAD

**Issue by aj0312** [common-voice/common-voice#3115]
**Describe the bug**
I am running Docker Desktop for Windows. When I run `docker-compose up` from project folder, the `web` container exits with error **`web        | /usr/bin/env: 'bash\r': No such file or directory`**

**To Reproduce**
1. Go to project folder, cloned from the repo
2. Run `docker-compose up`

**Expected behavior**
All 3 containers should start running

**Screenshots**
<img width="673" alt="docker-compose_up_windows_error" src="https://user-images.githubusercontent.com/19672287/118174058-fa4f9b00-b44b-11eb-9b13-a989a7ec0633.PNG">
<img width="389" alt="docker-compose_up_windows_error_containers_list" src="https://user-images.githubusercontent.com/19672287/118174067-fc195e80-b44b-11eb-827d-a9bcc835d69c.PNG">

**System/Software info:**
 - OS: Windows 10
 - Docker Desktop for Windows (using Linux containers)

**Comment by ftyers** [comment]
Looks like an issue with newlines in filenames. Try running it in Linux or changing your `\r\n` for `\n`? Although I have no idea if that will work.

**Comment by aj0312** [comment]
> 
> 
> Looks like an issue with newlines in filenames. Try running it in Linux or changing your `\r\n` for `\n`? Although I have no idea if that will work.

@ftyers  thanks for your suggestion. Yes, it seems newlines issue. But I am not sure either where to remove `\r` from. I checked config of `docker-compose.yaml`, `Dockerfile` and scripts of `prepare_s3.sh` and `wait-for-it.sh`, but in vain.

I had tried the mentioned steps in Linux containers and got the issue

**Comment by aj0312** [comment]
I found a solution, it's actually a workaround for the Windows docker I guess

I checked this answer and tried to do the same in **docker-compose.yaml** file

[https://stackoverflow.com/a/29045187/6814369](https://stackoverflow.com/a/29045187/6814369)

So I modified
`command: bash -c "/code/docker/wait-for-it.sh s3proxy:80 -- /code/docker/prepare_s3.sh && yarn && yarn start"`

with
`command: bash -c "sed $$'s/\r$$//' /code/docker/wait-for-it.sh > /code/docker/wait-for-it.sh && /code/docker/wait-for-it.sh s3proxy:80 -- /code/docker/prepare_s3.sh && yarn && yarn start"`

basically prepended with
sed $$'s/\r$$// 
to ignore \r in **wait-for-it.sh** file
